<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 244, 255)</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>220</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="manufacturer_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать производителя компьютера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="pc_model_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать модель компьютера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="os_name_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать имя ОС</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="os_version_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать версию ОС</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cpu_name_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать название процессора</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ram_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать объем ОЗУ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="gpu_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать модель видеокарты</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>200</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="ping_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Показать ping (задержку сигнала)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="upload_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Скорость загрузки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="download_box">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Скорость скачивания</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="choose_all_hardware_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Выбрать все</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="display_hardware_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Отобразить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="write_hardware_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Записать в файл</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>346</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(93, 89, 197);</string>
   </property>
   <property name="text">
    <string>Здесь вы можете посмотреть параметры своего ПК</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QPushButton" name="choose_all_internet_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Выбрать все</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="display_internet_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Отобразить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="write_internet_btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 140, 177);
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Записать в файл</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>12</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(93, 89, 197);</string>
   </property>
   <property name="text">
    <string>Здесь вы можете посмотреть скорость своего интернета</string>
   </property>
  </widget>
  <widget class="QListWidget" name="hardware_listwidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="internet_listwidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hardware_filename_lineedit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="internet_filename_lineedit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Имя файла для записи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Имя файла для записи</string>
   </property>
  </widget>
  <widget class="QLabel" name="hardware_error_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="internet_error_label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
